test/parallel_api/iterator/permutation_iterator_parallel_merge.pass.cpp

```diff
@@ -28,19 +28,88 @@ DEFINE_TEST_PERM_IT(test_merge, PermItIndexTag)
         ::std::iota(itBegin, itEnd, initVal);
     }
 
-    template <typename Policy, typename Iterator1, typename Iterator2, typename Iterator3, typename Size>
-    void
-    operator()(Policy&& exec,
-               Iterator1 first1, [[maybe_unused]] Iterator1 last1,
-               Iterator2 first2, [[maybe_unused]] Iterator2 last2,
-               Iterator3 first3, Iterator3 last3, Size n)
+    template <typename Policy, typename Size, typename Iterator3, typename TPermutationIterator>
+    struct TestImlementationLevel1
```

Comment:
```suggestion
    struct TestImplementationLevel1
```
Suggestion to update these cases (there are a few more).

Reply:
Fixed